ios/FluentUI/List/FluentListSection.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import SwiftUI
+
+/// This a wrapper around `SwiftUI.Section` that has fluent style applied. It is intended to be used in conjunction with `FluentUI.FluentList` and `FluentUI.ListItem`
+/// to provide a completely fluentized list, however, it can be used on it's own if desired.
+public struct FluentListSection<SectionContent: View, SectionHeaderContent: View, SectionFooterContent: View>: View {
+
+    // MARK: Initializer
+
+    /// Creates a `FluentListSection`
+    /// - Parameters:
+    ///   - content: content to show inside of the section.
+    ///   - header: content to show inside of the header. Defaults to an `EmptyView`.
+    ///   - footer: content to show inside of the footer. Defaults to an `EmptyView`.
+    public init(@ViewBuilder content: @escaping () -> SectionContent,
+                @ViewBuilder header: @escaping () -> SectionHeaderContent = { EmptyView() },
```

Comment:
how is spacing between sections affected if we provide a default "EmptyView()" for header and footer? I think it might add additional unwanted spacing.

Reply:
The spacing of the sections is identical if the `EmptyView()` is used as a default. I compared a `SwiftUI.Section` with a `FluentListSection` and both had the same dimensions/spacing.